### DISCUSSION TITLE
Transfer SOL/SPL Token after connect with web3modal over Solana blockchain

### DISCUSSION THREAD

**Discussion by juloxrox** [WalletConnect/walletconnect-monorepo#4479] [Developer Support]
Does anyone have a simple working example of connecting wallet with web3modal and then doing a transaction for sending SOL and SPL token to a receiver? I'm not using React and I don't find anything in vanilla JS..
Any help would be greatly appreciated! Thanks in advance

**Comment by outsmartchad** [comment]
My repo https://github.com/ManofDiligence/Solana-Memecoin-CLI has a command line tool for you to transfer spl token or sol to any Solana account
